src/styles/main.scss

```diff
@@ -1,3 +1,223 @@
-body {
+* {
   margin: 0;
 }
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+:root {
+  --blue: #00acdc;
+  --black: #000;
+  --grey: #616070;
+  --darkblue: #060b35;
+  --white: #fff;
+}
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0 41px;
+  padding-top: 51px;
+}
+
+.header {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  position: relative;
+  padding-right: 50px;
+
+  &__icon {
+    height: 40px;
+    width: 40px;
+    position: absolute;
+    left: 50px;
+    top: 10px;
+    bottom: 10px;
+  }
+
+  &__nav_list {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14px;
+    text-transform: uppercase;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+  }
+
+  &__nav_link {
+    text-decoration: none;
+    line-height: 60px;
+    color: var(--darkblue);
+    padding-top: 23px;
+    padding-bottom: 23px;
+  }
+
+  &__nav_link:hover {
+    color: var(--blue);
+  }
+
+  &__nav_item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__nav_link--is-active {
+    color: var(--blue);
+    position: relative;
+  }
+
+  &__nav_link--is-active::after {
+    content: "";
+    display: block;
+    width: 100%;
+    height: 4px;
+    position: absolute;
+    background: var(--blue);
+    border-radius: 8px;
+    bottom: 0;
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: var(--white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-right: 50px;
+  margin-bottom: 48px;
+  // height: 406px;
```

Comment:
remove commented code
follow to BEM metodology - blocks shouldn't have margins. instead create elements for .main block and set margins for card in element class